config/config_test.go

```diff
@@ -18,17 +17,22 @@ var repoMap = map[string]string{
 
 func TestLoad(t *testing.T) {
 	config, _ := Load(configYaml)
-	assert.Equal(t, repoMap, config.RepoMap)
+	if !reflect.DeepEqual(config.RepoMap, repoMap) {
```

Comment:
I still recommend `github.com/stretchr/testify` for this, since it also provides you with the fancy diff, though it'll:

1. Increase your dependency surface area
2. Make less obvious what's going on (depending on who you ask)